airflow/bin/cli.py

```diff
@@ -325,6 +325,21 @@ def task_state(args):
     print(ti.current_state())
 
 
+def dag_state(args):
+    """
+    Returns the state of a DagRun at the command line.
+
+    >>> airflow dag_state tutorial 2015-01-01
+    running
+    """
+    dag = get_dag(args)
+    dr = DagRun.find(dag.dag_id, execution_date=args.execution_date)
+    if len(dr) > 0:
```

Comment:
Also, not super keen on just using the first `dr` if len > 0. We expect to only ever get one, so I'm wondering if an Exception or logger.warn or something would be better if we see len > 1.